package.json

```diff
@@ -27,15 +27,21 @@
     "rss": "^1.2.2"
   },
   "devDependencies": {
+    "@sendgrid/mail": "^8.1.1",
```

Comment:
These somehow got added when I ran npm install. Clearly it's me, but not even sure how these got added since I didn't add them. Will revert.